src/index.d.ts

```diff
@@ -251,15 +251,29 @@ declare namespace RamdaAdjunct {
         isNotNumber(val: any): boolean;
 
         /**
-         * Checks if value is a positive `Number` primitive or object.
+         * Checks if value is a positive `Number` primitive or object. Zero is not considered
+         * negative.
          */
         isPositive(val: any): val is number;
 
         /**
-         * Checks if value is a negative `Number` primitive or object.
+         * Checks if value is a negative `Number` primitive or object. Zero is not considered
```

Comment:
Checks if value is a negative `Number` primitive or object. Zero is considered neither positive nor negative.